openff/toolkit/topology/molecule.py

```diff
@@ -2497,6 +2497,36 @@ def apply_elf_conformer_selection(
                 f"{type(toolkit_registry)}"
             )
 
+    def get_available_charge_methods(
```

Comment:
👍  Ah, I wouldn't have thought of it, but the Molecule API is a really smart place to let folks access this info.